lib/ecco/row_event_listener.rb

```diff
@@ -5,8 +5,11 @@ module Ecco
   class RowEventListener < EventListener
     ROW_EVENTS = [
       EventType::WRITE_ROWS,
+      EventType::EXT_WRITE_ROWS,
       EventType::UPDATE_ROWS,
+      EventType::EXT_UPDATE_ROWS,
       EventType::DELETE_ROWS,
+      EventType::EXT_DELETE_ROWS,
```

Comment:
> Shouldn't we have specs for both then? You added to the array here, but replaced them in the specs (e.g. WRITE_ROWS -> EXT_WRITE_ROWS)

Not sure it's worth it, since we need to stop and reconfigure MySQL to have multiple specs.

I'll probably refactor this and rely on https://github.com/shyiko/mysql-binlog-connector-java/blob/05bc0952c5de64fc836fcb9ca36f4b4570d21a72/src/main/java/com/github/shyiko/mysql/binlog/event/EventType.java#L191-L207 and add our own types that don't vary depending on the MySQL config and/or version.

But first step is just to get everything working with 5.6s default config.